filesystem/config.pan

```diff
@@ -5,8 +5,28 @@ unique template filesystem/config;
 
 include { 'quattor/functions/filesystem' };
 
+
+@{
+desc =  template customizing the default disk layout provided by the default configuration
+values = template namespace
+default = null
+required = no
+}
 variable FILESYSTEM_LAYOUT_CONFIG_SITE ?= null;
 
+@{
+desc =  template included at the beginning of the file system configuration and allowing to \
+ to redefine the default size of block devices defined in the default configuration
+values = template namespace
+default = FILESYSTEM_LAYOUT_CONFIG_SITE+'-init' or null if FILESYSTEM_LAYOUT_CONFIG_SITE is undefined
+required = no
+}
+variable FILESYSTEM_LAYOUT_CONFIG_INIT ?= if ( is_defined(FILESYSTEM_LAYOUT_CONFIG_SITE) ) {
+                                            if_exists(FILESYSTEM_LAYOUT_CONFIG_SITE + '-init');
+                                          } else {
+                                            null;
+                                          };
```

Comment:
That may be, but it's horrible, it makes all the lines far too long and makes them dependent on the length of the left hand side of the assignment, which when it changes causes the following lines to look like they have code changes.

To me, this is both easier to read and maintain.

```pan
variable FILESYSTEM_LAYOUT_CONFIG_INIT ?= if (is_defined(FILESYSTEM_LAYOUT_CONFIG_SITE)) {
    if_exists(FILESYSTEM_LAYOUT_CONFIG_SITE + '-init');
} else {
    null;
}; 
```

But I can't be bothered to argue the point any more and I certainly won't delay the PR for it.